Piece.java

```diff
@@ -0,0 +1,30 @@
+import javax.swing.text.Position;

```

Comment:
It doesn't look like this is what you want here. You can implement your own Position class including an x and y integer